content/automated-rust-releases.md

```diff
@@ -0,0 +1,582 @@
+    automated-rust-releases-aarch64-apple-darwin.tar.xz
+      [bin] automated-rust-releases
+      [misc] LICENSE-MIT
+      [checksum] automated-rust-releases-aarch64-apple-darwin.tar.xz.sha256
+    automated-rust-releases-x86_64-apple-darwin.tar.xz
+      [bin] automated-rust-releases
+      [misc] LICENSE-MIT
+      [checksum] automated-rust-releases-x86_64-apple-darwin.tar.xz.sha256
+    automated-rust-releases-x86_64-pc-windows-msvc.zip
+      [bin] automated-rust-releases.exe
+      [misc] LICENSE-MIT
+      [checksum] automated-rust-releases-x86_64-pc-windows-msvc.zip.sha256
+    automated-rust-releases-x86_64-unknown-linux-gnu.tar.xz
+      [bin] automated-rust-releases
+      [misc] LICENSE-MIT
+      [checksum] automated-rust-releases-x86_64-unknown-linux-gnu.tar.xz.sha256
+```
+
+And run `cargo dist build` if you actually want to build the artifacts. Also, if you set `pr-run-mode` to `upload` in the configuration, then the artifacts will be built and uploaded as `artifacts.zip` to the related commit on GitHub.
+
+<center>
+
+![cargo-dist artifacts](/cargo-dist-artifacts.png)
+
+</center>
+
+Now we are ready to build artifacts and release them on GitHub! There are a couple of steps left until we create a release and showcase everything.
+
+---
+
+### **Dependabot**
+
+[`Dependabot`](https://github.com/dependabot) is a tool built into GitHub for automatically updating project's dependencies via creating pull requests.
+
+We can simply configure it via creating `.github/dependabot.yml`:
+
+```yml
+version: 2
+updates:
+  # Maintain dependencies for Cargo
+  - package-ecosystem: cargo
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 10
+```
+
+Now `Dependabot` will start creating pull requests for Rust/GitHub Actions dependencies when they become out-of-date:
+
+<center>
+
+![dependabot PR](/dependabot-pr.png)
+
+[https://github.com/orhun/systeroid/pull/141](https://github.com/orhun/systeroid/pull/141)
+
+</center>
+
+---
+
+### **Mergify**
+
+> [`Mergify`](https://mergify.com/) is a CI/CD pipeline optimizer that manages your pull request, automates your GitHub workflow, and optimizes your CI costs.
+
+We will be using `Mergify` to:
+
+1. Automatically merge `Dependabot` pull requests.
+2. Update to the latest `main` branch for pull requests.
+
+Let's create the configuration in `.github/mergify.yml`:
+
+```yml
+pull_request_rules:
+  - name: Automatic merge for Dependabot pull requests
+    conditions:
+      - author=dependabot[bot]
+    actions:
+      merge:
+        method: squash
+
+  - name: Automatic update to the main branch for pull requests
+    conditions:
+      - -conflict # skip PRs with conflicts
+      - -draft # skip GH draft PRs
+      - -author=dependabot[bot] # skip dependabot PRs
```

Comment:
It would be nice to skip release-plz PRs as well here. Because release-plz force pushes to the PR.
Maybe you can do it by the branch of the PR? I.e. if it starts with `release-plz` (not `release-plz/` because it's going to change to `release-plz-` in the next release).

Reply:
I will wait until the release is available :D